README.md

```diff
@@ -56,6 +56,20 @@ well as providing a canonical history of what was deployed and when.
 
 ## Installation
 
+### Building
```

Comment:
We should probably add a note somewhere on testing too

Reply:
I think that'd make sense :)